misc/pytorch_toolkit/breast_ultrasound_simulation/src/exporter.py

```diff
@@ -0,0 +1,45 @@
+import torch
+from .model import GeneratorModel
+import argparse
+import os
+import subprocess
+
+def export_model_onnx(config_args):
+    device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
+    print(torch.cuda.is_available())
+    model_path = config_args['model_path']
+    gen_old = GeneratorModel(1).to(device)
+    state_dict = torch.load(model_path)
+    gen_old.load_state_dict(state_dict["generator1_weights"])
+    onnx_model = config_args['onnx_model_name']
+    dummy_input = torch.randn(1,1,142,142).cuda()
```

Comment:
In the dataset class you resize images to 128x128. Is there a typo or some more complicated logic behind that?